bidadjustment/build_rules.go

```diff
@@ -63,7 +63,7 @@ func merge(req *openrtb_ext.RequestWrapper, acct *openrtb_ext.ExtRequestPrebidBi
 	if extPrebid == nil && acct == nil {
```

Comment:
Thoughts on changing this logic? I'm thinking it might be easier to understand if it were like this:
```
if (extPrebid == nil || extPrebid.BidAdjustments == nil) {
	if acct == nil {
		return nil, nil	
	}
	return acct, nil
}
if acct == nil {
	return extPrebid.BidAdjustments, nil
}
```

Reply:
```
	if extPrebid == nil || extPrebid.BidAdjustments == nil {
		return acct, nil
	}

	if acct == nil {
		return extPrebid.BidAdjustments, nil
	}
```